pkg/debian/salt-master.preinst

```diff
@@ -1,47 +1,67 @@
 #!/bin/sh
+# preinst script for salt-master.
+#
+# See: dh_installdeb(1).
 
-. /usr/share/debconf/confmodule
+set -e
+
+# Summary of how this script can be called:
+#        * <new-preinst> 'install'
+#        * <new-preinst> 'install' <old-version>
+#        * <new-preinst> 'upgrade' <old-version>
+#        * <old-preinst> 'abort-upgrade' <new-version>
+# for details, see https://www.debian.org/doc/debian-policy/ or
+# the debian-policy package.
+
+[ -n "$SALT_HOME" ] || SALT_HOME=/opt/saltstack/salt
+[ -n "$SALT_USER" ] || SALT_USER=salt
+[ -n "$SALT_NAME" ] || SALT_NAME="Salt"
+[ -n "$SALT_GROUP" ] || SALT_GROUP=salt
 
 case "$1" in
   install)
-    [ -z "$SALT_HOME" ] && SALT_HOME=/opt/saltstack/salt
-    [ -z "$SALT_USER" ] && SALT_USER=salt
-    [ -z "$SALT_NAME" ] && SALT_NAME="Salt"
-    [ -z "$SALT_GROUP" ] && SALT_GROUP=salt
     PY_VER=$(/opt/saltstack/salt/bin/python3 -c "import sys; sys.stdout.write('{}.{}'.format(*sys.version_info)); sys.stdout.flush();")
 
     # Reset permissions to fix previous installs
```

Comment:
I'd suggest removing this find/chmod entirely - the original purpose of this was broken in #66218 - the purpose was to reset these directories to being owned by root:root so that only the minimum set of dirs that needed to be owned by the salt user would be set in the postinst

This is essentially a no-op currently, it finds files/dirs owned by salt or group salt and chowns them to salt:salt.